target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -73,6 +95,18 @@ define Device/ubnt_nano-m
 endef
 TARGET_DEVICES += ubnt_nano-m
 
+define Device/ubnt_nanostationacl
+  $(Device/ubnt-wa)
+  DEVICE_TITLE := Ubiquiti Nanostation AC loco
+  DEVICE_PACKAGES += kmod-ath10k ath10k-firmware-qca988x
+  IMAGE_SIZE := 15744k
+  IMAGES := factory.bin sysupgrade.bin
```

Comment:
`IMAGES += factory.bin` is enough. The sysupgrade.bin is already set as to be build image in [ath79 toplevel image Makefile](https://github.com/openwrt/openwrt/blob/aed46928fa5a0cd9df076cb4d11c76a8413cdae4/target/linux/ath79/image/Makefile#L64).

Reply:
Yep should have looked at the Makefile